common/v2/features/DeFiZap/components/DashboardZapCTA.tsx

```diff
@@ -61,17 +46,10 @@ const DashboardZapCTA = ({ className }: any) => {
     );
   };
 
-  const RightHeading = (
-    <DefiZapLogo>
-      <DefiZapLogoImage src={defizaplogo} />
-      <DefiZapLogoText>{headingRight}</DefiZapLogoText>
-    </DefiZapLogo>
-  );
-
   return (
     <DashboardPanel
       heading={'Put your ETH to work for you'}
-      headingRight={RightHeading}
+      headingRight={DeFiZapLogo()}
```

Comment:
Can we use it as a tag instead? Just seems prettier 😄

Reply:
> Can we use it as a tag instead? Just seems prettier smile

What do you mean by "tag"?